lib/src/sql/statements/remove/table.rs

```diff
@@ -13,6 +13,7 @@ use std::fmt::{self, Display, Formatter};
 #[revisioned(revision = 1)]
 pub struct RemoveTableStatement {
 	pub name: Ident,
+	pub if_exists: bool,
```

Comment:
Since this is a new field the revision needs to be changed.
```suggestion
#[revisioned(revision = 2)]
pub struct RemoveTableStatement {
	pub name: Ident,
	#[revision(start = 2)]
	pub if_exists: bool,
```